config/routes.rb

```diff
@@ -2,5 +2,7 @@
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
     get "/help", to: "static_pages#help"
+    get "/signup", to: "users#new"
+    resources :users, only: %i(new create show)
```

Comment:
Đã dùng router của cái users/new thành signup rồi thì sửa thành thế này thôi e
```suggestion
    resources :users, only: %i(create show)
```